src/portable/nuvoton/nuc505/dcd_nuc505.c

```diff
@@ -300,7 +297,15 @@ void dcd_set_config(uint8_t rhport, uint8_t config_num)
 {
   (void) rhport;
   (void) config_num;
-  configuration_changed = true;
+
+  /* reset context of all non-control endpoints */
+
+  for (enum ep_enum ep_index = PERIPH_EPA; ep_index < PERIPH_MAX_EP; ep_index++)
```

Comment:
since we are removing `dcd_set_config()`, we should not implement it here. Don't worry about resetting data toggle, it is enough to do that in dcd_edpt_open(). Since SetConfig() will follows by an series of endpoints open (non-zero) or close_all (zero).